src/main/java/trivia/Board.java

```diff
@@ -21,4 +21,17 @@ public static Board create(QuestionCategoryName... categories) {
   public QuestionCategoryName getCategoryName(int position) {
     return positionToCategory[position];
   }
+
+  public void movePlayer(Player player, int roll) {
```

Comment:
Moving a player is joint effort of Player and Board.
Board knows how many places are on the board, and when to start iterating from zero,
but Player knows his own position on the board.

It's tricky to accomplish that Player doesn't depend on Board internal details and vice versa,
so it looks like this should be in GameBetter itself.

GameBetter should have **move** method, which reads TOTAL_PLACES from Board and sets new position in Player with simple setter method.

Reply:
What if the board keeps track of where each player is, i.e. we move that responsibility to the board?

Reply:
You'll find this answered in one of a previous comments.